### DISCUSSION TITLE
[Autoform] Setting default value for date picker

### DISCUSSION THREAD

**Discussion by AGgup8** [unovue/shadcn-vue#798] [Q&A]
Does anyone know how to set a default date value for an autoform?

The docs say "If you want to set default value of date, convert it to Date first using new Date(val)", 

`const formSchema = z.object({
  orderDate: z.coerce
    .date()
    .default(new Date())
    .describe('Order Date'),
})`

However, it causes a console error because the AutoFormFieldDate expects a DateValue value type (from the @internationalzed/date package).

`TypeError: slotProps.componentField.modelValue.toDate is not a function`

**Comment by HipyCas** [comment]
I've found a solution in issue #780 ([this comment](https://github.com/unovue/shadcn-vue/issues/780#issuecomment-2499996989))
